models/users.model.js

```diff
@@ -1,3 +1,12 @@
 const db = require("../db/connection");
 
-//Task 10: user model
\ No newline at end of file
+//TASK 10
+exports.selectUsers = () => {
+    return db.query("SELECT * FROM users;")
+    .then(( { rows }) => {
+        if (rows.length === 0) {
+            return Promise.reject({ status: 404, msg: "No users found"});
+        }
```

Comment:
You have no test to check this behaviour - and you will never hit it because the only reason you would get no users would be a database issue which would give you a 500 response anyway. The check is not needed :)